components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/StorageFacetImpl.java

```diff
@@ -280,7 +285,7 @@ private StorageTx openStorageTx(final ODatabaseDocumentTx db, final boolean isUs
     return StateGuardAspect.around(
         new StorageTxImpl(
             createdBy(),
-            new BlobTx(blobStore),
+            new BlobTx(localNodeAccess.getId(), blobStore),
```

Comment:
maybe slightly saner to pass in the localNodeAcess and fetch the id when needed closer to the blobref creation.